front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -0,0 +1,194 @@
+  DynamicDataSheetGrid,
+} from 'react-datasheet-grid';
+import { useTranslation } from 'react-i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import { useOsrdConfActions } from 'common/osrdContext';
+import { isVia } from 'modules/pathfinding/utils';
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+import type { PathStep } from 'reducers/osrdconf/types';
+import { useAppDispatch } from 'store';
+
+import timeColumn from './TimeColumnComponent';
+
+type TimesStopsProps = {
+  pathProperties: ManageTrainSchedulePathProperties;
+  pathSteps?: (PathStep | null)[];
+  setPathProperties: (pathProperties: ManageTrainSchedulePathProperties) => void;
+};
+
+type PathWaypointColumn = SuggestedOP & {
+  isMarginValid: boolean;
+};
+
+const marginRegExValidation = /^\d+(\.\d+)?%$|^\d+(\.\d+)?min\/km$/;
+
+const formatSuggestOPsToRowOPs = (
+  operationalPoints: SuggestedOP[],
+  t: TFunction<'timesStops', undefined>
+): PathWaypointColumn[] => {
+  const rowOPs = operationalPoints?.map((op) => ({
+    ...op,
+    name: op.name || t('waypoint', { id: op.opId }),
+    isMarginValid: op.theoreticalMargin ? marginRegExValidation.test(op.theoreticalMargin) : true,
+    onStopSignal: op.onStopSignal || false,
+  }));
+  return rowOPs;
+};
+
+const deleteViaButton = (removeVia: () => void, isRowVia: boolean) => {
+  if (isRowVia) {
+    return (
+      <button type="button" onClick={removeVia}>
+        ❌
+      </button>
+    );
+  }
+  return <div />;
+};
+
+const TimesStops = ({ pathProperties, pathSteps = [], setPathProperties }: TimesStopsProps) => {
+  const { t } = useTranslation('timesStops');
+  const dispatch = useAppDispatch();
+  const { upsertViaFromSuggestedOP, updatePathSteps } = useOsrdConfActions();
+
+  const [timesStopsSteps, setTimesStopsSteps] = useState<PathWaypointColumn[]>([]);
+
+  useEffect(() => {
+    const suggestedOPs = formatSuggestOPsToRowOPs(pathProperties.allVias, t);
+    setTimesStopsSteps(suggestedOPs);
+  }, [pathSteps]);
+
+  const columns: Column<PathWaypointColumn>[] = useMemo(
+    () => [
+      {
+        ...keyColumn<PathWaypointColumn, 'name'>('name', createTextColumn<string | undefined>()),
+        title: t('name'),
+        disabled: true,
+      },
+      {
+        ...keyColumn<PathWaypointColumn, 'ch'>('ch', createTextColumn()),
+        title: 'Ch',
+        disabled: true,
+        grow: 0.1,
+      },
+      {
+        ...keyColumn<PathWaypointColumn, 'arrival'>('arrival', timeColumn),
+        title: t('arrival_time'),
+        cellClassName: ({ rowIndex }) => (rowIndex === 0 ? 'dsg-hidden-cell' : ''),
+        grow: 0.6,
+      },
+      {
+        ...keyColumn<PathWaypointColumn, 'stop_for'>(
+          'stop_for',
+          createTextColumn({
+            continuousUpdates: false,
+            alignRight: true,
+          })
+        ),
+        title: `${t('stop_time')} (s)`,
```

Comment:
Move the ' (s)' in the translations
```suggestion
        title: t('stop_time'),
```

Reply:
Done